.travis.yml

```diff
@@ -57,3 +56,27 @@ jobs:
           branch: master
           repo: plus3it/tardigrade-ci
           condition: '"$PRIOR_VERSION" != "$RELEASE_VERSION"'
+
+    - stage: deploy
+      name: Deploy docker image to dockerhub
+      if: (branch = master OR tag is present) AND type != pull_request AND repo = plus3it/tardigrade-ci
+      script: docker build --build-arg GITHUB_ACCESS_TOKEN=$GITHUB_ACCESS_TOKEN -t "$IMAGE_NAME" -f Dockerfile .
+      before_deploy:
+        - export DOCKERHUB_TARGET="plus3it/tardigrade-ci:${TRAVIS_TAG:-latest}"
+        - docker tag "$IMAGE_NAME" "$DOCKERHUB_TARGET"
+        - echo Pushing image "$DOCKERHUB_TARGET"
+        - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+      after_deploy: echo Pushed image "$DOCKERHUB_TARGET"
+      deploy:
+        - provider: script
+          # push "latest" to dockerhub
+          script: docker push "$DOCKERHUB_TARGET"
+          skip_cleanup: true
+          on:
+            branch: master
```

Comment:
Reliance on "master" will be problematic when "main" branches start to be used.

Reply:
Indeed, we had to deal with that in watchmaker when we changed the default branch to "main". Thankfully, that change is still on our own timeframe and no-one else's. This project uses "master" currently, so the travis-ci references need to be for "master" also.